data/sql/updates/pending_db_world/Flame-Arrows.sql

```diff
@@ -0,0 +1,24 @@
+
+DELETE FROM `creature` WHERE `id1` = 17461;
+DELETE FROM `creature` WHERE `id1` = 20923;
+INSERT INTO `creature` VALUES (151283, 20923, 0, 0, 540, 0, 0, 2, 1, 1, 512.687, 315.652, 2.0405, 2.98451, 86400, 0, 0, 103320, 0, 0, 0, 0, 0, '', 0);
+INSERT INTO `creature` VALUES (151118, 17461, 0, 0, 540, 0, 0, 1, 1, 1, 512.687, 315.652, 2.0405, 2.98451, 86400, 0, 0, 1, 0, 0, 0, 0, 0, '', 46924);

```

Comment:
1. Always try to include more than one column when deleting entries, especially in the creature table.
2. Try to add the column names as well. It's unlikely but in the case the table changes structure it's useful to keep the names.
3. Shorten the queries when possible, 1 query runs faster than 10 😛 
```suggestion
DELETE FROM `creature` WHERE `id1` IN (17461, 20923) AND `guid` IN (151118, 151283);
INSERT INTO `creature` (`guid`, `id1`, `id2`, `id3`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `ScriptName`, `VerifiedBuild`) VALUES 
(151283, 20923, 0, 0, 540, 0, 0, 2, 1, 1, 512.687, 315.652, 2.0405, 2.98451, 86400, 0, 0, 0, 0, 0, 0, 0, 0, '', 46924),
(151118, 17461, 0, 0, 540, 0, 0, 1, 1, 1, 512.687, 315.652, 2.0405, 2.98451, 86400, 0, 0, 0, 0, 0, 0, 0, 0, '', 46924);

```

Reply:
ok